applications/redeye-e2e/src/integration/e2e/redteam/timeline.cy.js

```diff
@@ -14,8 +14,8 @@ describe('Timeline tests', () => {
 		cy.selectCampaign(camp);
 
 		// Log the starting position of the timeline bar
-		cy.get('[cy-test=timeline-animated-line]')
-			.invoke('attr', 'x1')
+		cy.get('[cy-test=timeline-scrubber]')
+			.invoke('attr', 'x1') // @QA - we are now moving the scrubber using `style="transform:translateX(00px)"`, not `x1` & `x2`
```

Comment:
@sang2925 && || @ccarpenter28,
I've updated the scrubber referenced in this test to change its position using transform rather than the `x1` and `x2` attributes.